.github/workflows/integration_tests.yml

```diff
@@ -250,34 +253,47 @@ jobs:
         if: env.CLOUD_PROVIDER == 'GKE'
         env:
           GKE_VERSION: ${{ matrix.PLATFORM_VERSION }}
-          GCLOUD_SERVICE_KEY: ${{ secrets.GCLOUD_SERVICE_KEY }}
+          GCLOUD_SERVICE_KEY: ${{ secrets.GCLOUD_SERVICE_KEY_NEW }}
```

Comment:
```suggestion
          GCLOUD_SERVICE_KEY: ${{ secrets.GCLOUD_RESTRICTED_SERVICE_KEY }}
```

Reply:
I named it differently so that both keys can live in the repo secrets for now

Reply:
renamed it now